### ISSUE TITLE
Argument 1 passed to RegexHelper::isEscapable() must be of the type string, null given

### ISSUE THREAD

**Issue by ntzm** [thephpleague/commonmark#403]
**Version(s) affected**: 1.2.1

**Description**

When converting malformed markdown, I get a TypeError

**How to reproduce**  

Markdown:
```md
[](\B\
```

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to League\CommonMark\Util\RegexHelper::isEscapable() must be of the type string, null given, called in /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Util/LinkParserHelper.php on line 101 and defined in /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Util/RegexHelper.php:74
Stack trace:
#0 /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Util/LinkParserHelper.php(101): League\CommonMark\Util\RegexHelper::isEscapable(NULL)
#1 /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Util/LinkParserHelper.php(41): League\CommonMark\Util\LinkParserHelper::manuallyParseLinkDestination(Object(League\CommonMark\Cursor))
#2 /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Inline/Parser/CloseBracketParser.php(154): League\CommonMark\Util\LinkParserHelper::parseLinkDestination(Object(League\CommonMark\Cursor))
#3 /home/nat/Code/personal/fuzzer-test/commo in /home/nat/Code/personal/fuzzer-test/commonmark/vendor/league/commonmark/src/Util/RegexHelper.php on line 74
```

**Comment by GrahamCampbell** [comment]
https://github.com/thephpleague/commonmark/pull/404 should fix your issue. ;)

**Comment by colinodell** [comment]
Fixed via https://github.com/thephpleague/commonmark/commit/db580f692f2f324b745276b1fb7e27f453738e34 (thanks Graham!) and included in new releases 1.1.3 and 1.2.2.
